cbmex/cbmex.cpp

```diff
@@ -22,6 +22,7 @@
 
 */
 
+#include <algorithm>  // Use C++ default min and max implementation.
 #include "StdAfx.h"

```

Comment:
OK I can fix that. Do you want me to do a rebase and force push, or just add on another commit?